src/goto-programs/goto_function.h

```diff
@@ -16,29 +16,23 @@ Date: May 2018
 
 #include <iosfwd>
 
-#include <util/deprecate.h>
 #include <util/find_symbols.h>
 #include <util/std_types.h>
 
 #include "goto_program.h"
 
-/// A goto function, consisting of function type (see #type), function body (see
-/// #body), and parameter identifiers (see #parameter_identifiers).
+/// A goto function, consisting of function body (see #body) and parameter
+/// identifiers (see #parameter_identifiers).
 class goto_functiont
 {
 public:
   goto_programt body;
 
-  /// The type of the function, indicating the return type and parameter types
-  DEPRECATED(SINCE(2019, 2, 16, "Get the type from the symbol table instead"))
-  code_typet type;
-
   typedef std::vector<irep_idt> parameter_identifierst;
-
   /// The identifiers of the parameters of this function
   ///
   /// Note: This is now the preferred way of getting the identifiers of the
-  /// parameters. The identifiers in the type will go away.
+  /// parameters. The identifiers in the type are now gone.
```

Comment:
:smile: nice!

Reply:
I am afraid they are still there. "The type" here refers to the type in the symbol table.

Reply:
Err, true. Change to the comment reverted/dropped.